File: PYTHON/tiket.py
from film import Film

class Tiket(Film):
    def __init__(self, media_id=0, judul="", tahun=0, hak_cipta="", genre="", durasi_menit=0,
                 rumah_produksi="", nama_tempat="", studio="", harga=0, row='', seat=0, tanggal="", waktu=""):
        super().__init__(media_id, judul, tahun, hak_cipta, genre, durasi_menit, rumah_produksi)
        self._nama_tempat = nama_tempat
        self._studio = studio
        self.set_harga(harga)
        self._row = row
        self._seat = seat
        self._tanggal = tanggal
        self._waktu = waktu

    def get_nama_tempat(self): return self._nama_tempat
    def set_nama_tempat(self, nama_tempat): self._nama_tempat = nama_tempat

    def get_studio(self): return self._studio
    def set_studio(self, studio): self._studio = studio

    def get_harga(self): return self._harga
    def set_harga(self, harga): self._harga = max(harga, 0)

    def get_row(self): return self._row
    def set_row(self, row): self._row = row

    def get_seat(self): return self._seat
    def set_seat(self, seat): self._seat = seat

    def get_tanggal(self): return self._tanggal
    def set_tanggal(self, tanggal): self._tanggal = tanggal

    def get_waktu(self): return self._waktu
    def set_waktu(self, waktu): self._waktu = waktu

File: PHP/tiket.php
<?php

require_once 'Film.php';

class Tiket extends Film {
    private string $namaTempat;
    private string $studio;
    private int $harga;
    private string $row;
    private int $seat;
    private string $tanggal;
    private string $waktu;

    public function __construct(
        int $id = 0, 
        string $judul = "", 
        int $tahun = 0, 
        string $hakCipta = "", 
        string $genre = "", 
        int $durasiMenit = 0, 
        string $rumahProduksi = "",
        string $namaTempat = "", 
        string $studio = "", 
        int $harga = 0, 
        string $row = "", 
        int $seat = 0, 
        string $tanggal = "", 
        string $waktu = ""
    ) {
        parent::__construct($id, $judul, $tahun, $hakCipta, $genre, $durasiMenit, $rumahProduksi);
        $this->namaTempat = $namaTempat;
        $this->studio = $studio;
        $this->setHarga($harga);
        $this->row = $row;
        $this->seat = $seat;
        $this->tanggal = $tanggal;
        $this->waktu = $waktu;
    }

    public function getNamaTempat(): string { return $this->namaTempat; }
    public function setNamaTempat(string $namaTempat): void { $this->namaTempat = $namaTempat; }

    public function getStudio(): string { return $this->studio; }
    public function setStudio(string $studio): void { $this->studio = $studio; }

    public function getHarga(): int { return $this->harga; }
    public function setHarga(int $harga): void { $this->harga = max($harga, 0); }

    public function getRow(): string { return $this->row; }
    public function setRow(string $row): void { $this->row = $row; }

    public function getSeat(): int { return $this->seat; }
    public function setSeat(int $seat): void { $this->seat = $seat; }

    public function getTanggal(): string { return $this->tanggal; }
    public function setTanggal(string $tanggal): void { $this->tanggal = $tanggal; }

    public function getWaktu(): string { return $this->waktu; }
    public function setWaktu(string $waktu): void { $this->waktu = $waktu; }
}